modules/options-for-notebook-server-environments.adoc

```diff
@@ -292,12 +352,27 @@ a| * Python 3.8
 * Elyra 3.15
 
 ifndef::upstream[]
-| code-server (Technology Preview)
+.2+| code-server (Technology Preview)
 endif::[]
 ifdef::upstream[]
-| code-server
+.2+| code-server
 endif::[]
-| 2023.2 (Recommended)
+| 2024.1 (Recommended)
+a| * Python 3.9
+* Boto3 1.29
+* Kafka-Python 2.0
+* Matplotlib 3.8
+* Numpy 1.26
+* Pandas 2.1
+* Plotly 5.18
+* Scikit-learn 1.3
+* Scipy 1.11
+* Sklearn-onnx 1.15
+* Ipykernel 6.26
+* (code-server plugin) Python 2024.2.1
+* (code-server plugin) Jupyter 2023.9.100
```

Comment:
These plugin versions are truly correct? :thinking: if so we should check that it's okay to mix these versions (2024 vs 2023)

Reply:
According to the info provided, yes. Should this be checked with the eng team?

Reply:
Confirmed this is correct as-is.